ios/Errors.swift

```diff
@@ -33,6 +33,25 @@ enum CreateTokenErrorType: String {
 }
 
 class Errors {
+    static internal let isPIClientSecretValidRegex: NSRegularExpression? = try? NSRegularExpression(
```

Comment:
I think it also makes sense to add one for `customerEphemeralKeySecret` and check that it starts with `ek_`?